apps/schematic/api/Dockerfile

```diff
@@ -1,41 +1,82 @@
-FROM python:3.10.12-slim-buster
+FROM tiangolo/uwsgi-nginx-flask:python3.10
 
-ENV APP_DIR=/opt/app
+# add label 
+LABEL maintainer="Milen Nikolov, Andrew Lamb, Mialy DeFelice, Gianna Jordan, Lingling Peng"
```

Comment:
Use the opencontainer standard for labels: https://github.com/opencontainers/image-spec/blob/main/annotations.md

Reply:
Thanks Thomas. I tried reading through the documentation but I got a bit confused about what the "standard" format needs to be (in the documentation, they don't have any examples other than "contact details of the people or organization responsible for the image (freeform string)"). Is the standard something like: 
```
LABEL maintainer="A <email address of A>, B <email address of B>,"
```

Reply:
Example (check value format from spec):

```
LABEL org.opencontainers.image.authors='airship-discuss@lists.airshipit.org, irc://#airshipit@freenode' \
      org.opencontainers.image.url='https://airshipit.org' \
      org.opencontainers.image.documentation='https://airship-armada.readthedocs.org' \
      org.opencontainers.image.source='https://opendev.org/airship/armada' \
      org.opencontainers.image.vendor='The Airship Authors' \
      org.opencontainers.image.licenses='Apache-2.0'
```

Reply:
Thanks Thomas. I changed the label to: 
```
LABEL org.opencontainers.image.authors= 'A <email address of A>, B <email address of B>'
```

Reply:
Thank you for adopting this standard labels. Yes, looks good to me. One potential improvement would be to split the label value over multiple lines. I'm not sure if our linter (hadolint) cares about this. I wasn't sure that it was actually possible but it looks like it is using this format based on this [article](https://dockerlabs.collabnix.com/beginners/dockerfile/Label_instruction.html):

```
LABEL description="This text illustrates \
that label-values can span multiple lines."
```

I'm resolving this comment now.